modules/nw-kuryr-migration-about.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_content-type: CONCEPT
+[id="nw-kuryr-ovn-kubernetes-migration-about_{context}"]
+= Migration to the OVN-Kubernetes network provider
+
+You can manually migrate a cluster that runs on {rh-openstack-first} to the OVN-Kubernetes network provider. During the migration process, your cluster will be unreachable for a brief time. 
+
+[IMPORTANT]
+====
+Migration to OVN-Kubernetes is a one-way process.
+
+During migration, your cluster will be unreachable for a brief time. 
+====
+
+[id="considerations-kuryr-migrating-network-provider_{context}"]
+== Considerations when migrating to the OVN-Kubernetes network provider
+
+Kubernetes namespaces are kept by Kuryr in separate {rh-openstack} networking service (Neutron) subnets. Those subnets and the IP addresses that are assigned to individual pods are not preserved during the migration.
+
+[id="how-the-kuryr-migration-process-works_{context}"]
+== How the migration process works
+
+The following table summarizes the migration process by relating the steps that you perform in the process with the actions that your cluster and Operators take.
+
+.The Kuryr to OVN-Kubernetes migration process
+[cols="1,1a",options="header"]
+|===
+
+|User-initiated steps|Migration activity
+
+|
+Set the `migration` field of the `Network.operator.openshift.io` custom resource (CR) named `cluster` to `OVNKubernetes`. Verify that the value of the `migration` field is `null` before setting it to another value.
+|
+Cluster Network Operator (CNO):: Updates the status of the `Network.config.openshift.io` CR named `cluster` accordingly.
+Machine Config Operator (MCO):: Deploys an update to the systemd configuration that is required by OVN-Kubernetes. By default, the MCO updates a single machine per pool at a time. As a result, large clusters have longer migration times.
+
+|Update the `networkType` field of the `Network.config.openshift.io` CR.
+|
+CNO:: Performs the following actions:
++
+--
+* Destroys the Kuryr control plane pods, which means Kuryr CNIs and the Kuryr controller.
```

Comment:
I have altered this sentence from "including Kuryr CNIs and the Kuryr controller." to "which means Kuryr CNIs and the Kuryr controller.", as controller and CNIs are what we call pods of Kuryr control plane.

Reply:
```suggestion
* Destroys the Kuryr control plane pods: Kuryr CNIs and the Kuryr controller.
```